Src/libCZI/CreateBitmap.cpp

```diff
@@ -46,7 +46,7 @@ static std::shared_ptr<libCZI::IBitmapData> CreateBitmapFromSubBlock_Uncompresse
 
     const auto& sbBlkInfo = subBlk->GetSubBlockInfo();
 
-    // TODO: How exactly shoud the stride be derived? It seems that stride must be exactly linesize.
+    // The stride with an uncompressed bitmap in CZI is exactly the linesize.
```

Comment:
As far as I remember, the documentation does not mention this at all. However, I arrived at this conclusion by reverse engineering. What's for sure - the stride **must** be defined implicitly (or: given by some convention) since it is nowhere given in the data. And, I'd say "minimal stride" is not a bad convention, it's easy and it's space efficient.
Note to myself - when revising the documentation, make sure that this is stated.